src/Serializers/ClaimsSerializer.php

```diff
@@ -20,7 +20,7 @@ class ClaimsSerializer implements DispatchableSerializer {
 	/**
 	 * @var Serializer
 	 */
-	private $claimSerializer;
+	private $statementSerializer;
```

Comment:
A full text search in the code base always brings these "false" positives up. Increasing code quality even in deprecated code does help me and others. I think this is worth it.